pkg/setting/setting_unified_storage.go

```diff
@@ -0,0 +1,28 @@
+package setting
+
+import (
+	"strings"
+
+	"github.com/grafana/grafana/pkg/apiserver/rest"
+)
+
+func (cfg *Cfg) setUnifiedStorageConfig() {
```

Comment:
Can you add a comment here for what the settings look like in config.ini?

Currently I have:
```
[grafana-apiserver]
storage_type = unified ; 

; Dual write settings
dashboards.dashboard.grafana.app=1
playlists.playlist.grafana.app=2
```

Reply:
so, config.ini looks like this:
```
[unified_storage.playlists.playlist.grafana.app]
dualWriterMode = 2
```
we can add more fields in each section. That way we can change each config per resource.